components/compliance-service/ingest/server/compliance.go

```diff
@@ -197,14 +203,39 @@ func (s *ComplianceIngestServer) handleNotifications(ctx context.Context, report
 	return nil
 }
 
-func (s *ComplianceIngestServer) sendNodeInfoToManager(ctx context.Context, node manager.NodeMetadata) error {
+func (s *ComplianceIngestServer) sendNodeInfoToManager(ctx context.Context, node manager.NodeMetadata, profiles []*ingest_inspec.Profile) error {
 	if s.mgrClient == nil {
 		return fmt.Errorf("no manager client found")
 	}
+	node.ScanData.Status = getReportStatus(profiles)
 	logrus.Debugf("sendNodeInfoToManager handing-over node to manager %+v", s)
 	_, err := s.mgrClient.ProcessNode(ctx, &node)
 	if err != nil {
 		return errors.Wrap(err, "sendNodeInfoToManager error calling ProcessNode")
 	}
 	return nil
 }
+
+func getReportStatus(profiles []*ingest_inspec.Profile) (status string) {
+	// start with a status of passed as the default status
+	status = "passed"
+	skippedCounter := 0
+	for _, profile := range profiles {
+		// if any profile is failed, report is failed
+		if profile.Status == "failed" {
+			status = "failed"
+			break
+		}
+		// if all profiles are skipped, the report is skipped
+		// so we keep a count of skipped profiles and later
+		// check if the amount of profiles matches the skipped counter,
+		// setting the status to skipped if they match.
+		if profile.Status == "skipped" {
+			skippedCounter++
+		}
+	}
+	if skippedCounter == len(profiles) {
+		status = "skipped"
+	}
+	return
+}
```

Comment:
There's a linter that complains about the named return value (or whatever this is called). We've turned it off for compliance-service but we'll probably want to enable it eventually.